src/arrayReverse.test.js

```diff
@@ -8,13 +8,16 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(Array.isArray(arrayReverse(['Mate', 'Academy']))).toBeTruthy();
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse([''])).toEqual(['']);
   });
 
-  // write more tests here
+  it(`should return correct strings length`, () => {
+    expect(arrayReverse(['I', 'am', 'a', 'student!']))
+      .toEqual(['!', 'tn', 'e', 'dutsamaI']);
```

Comment:
The expected output in this test case seems incorrect. If the function is supposed to reverse the array, the expected result should be ['student!', 'a', 'am', 'I']. If the function is intended to reverse each string, the expected result should be ['I', 'ma', 'a', '!tneduts']. Please clarify the intended behavior and adjust the test case accordingly.